src/address-manager/index.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const debug = require('debug')
+const log = debug('libp2p:addresses')
+log.error = debug('libp2p:addresses:error')
+
+const multiaddr = require('multiaddr')
+
+/**
+ * Responsible for managing the peer addresses.
```

Comment:
Clarification of whos addresses
```suggestion
 * Responsible for managing this peers addresses.
```